src/xlsx.js

```diff
@@ -21,27 +21,28 @@ export class Workbook {
   }
 }
 
-function extract(sheet, {range, headers = false} = {}) {
+function extract(sheet, {range, headers} = {}) {
```

Comment:
Since we only care whether _headers_ is truthy, there’s no advantage to a default value.